x-pack/legacy/plugins/siem/cypress/integration/lib/inspect/selectors.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HOSTS_PAGE, HOSTS_PAGE_TAB_URLS, NETWORK_PAGE } from '../urls';
+
+export const INSPECT_BUTTON_ICON = '[data-test-subj="inspect-icon-button"]';
+export const INSPECT_MODAL = '[data-test-subj="modal-inspect-euiModal"]';
+export const TIMELINE_SETTINGS_ICON = '[data-test-subj="settings-gear"]';
+export const TIMELINE_INSPECT_BUTTON = '[data-test-subj="inspect-empty-button"]';
+
+export const INSPECT_BUTTONS_IN_SIEM = [
```

Comment:
Some of the entries in this array have a `type` property, while others do not.

Consider adding an`interface` to the definition:

```
interface InspectButtonMetadata {
  id: string;
  title: string;
  type?: string;
  url: string;
}

export const INSPECT_BUTTONS_IN_SIEM: InspectButtonMetadata[] = [
```

, which would ensure new entries have all the required properties, and perhaps make `type` mandatory, if that sounds reasonable.